lib/core/cardano-wallet-core.cabal

```diff
@@ -217,11 +219,12 @@ test-suite unit
       Cardano.Wallet.DummyTarget.Primitive.Types
       Cardano.Wallet.Network.BlockHeadersSpec
       Cardano.Wallet.NetworkSpec
-      Cardano.Wallet.Primitive.AddressDerivation.RandomSpec
-      Cardano.Wallet.Primitive.AddressDerivation.SequentialSpec
+      Cardano.Wallet.Primitive.AddressDerivation.ByronSpec
+      Cardano.Wallet.Primitive.AddressDerivation.ShelleySpec
       Cardano.Wallet.Primitive.AddressDerivationSpec
       Cardano.Wallet.Primitive.AddressDiscovery.RandomSpec
       Cardano.Wallet.Primitive.AddressDiscovery.SequentialSpec
+      Cardano.Wallet.Primitive.AddressDiscoverySpec
```

Comment:
This is where most of the `Cardano.Wallet.Jormungandr.CompatibilitySpec` specs have been moved.